### PR TITLE
add new cyborg laws

### PR DESCRIPTION
**Pull request by DangerRevolution**
## About the PR

Adds:
- Pranksimov
- Royalty
- Aggressive

## Why / Balance

More variety

Also updates the comment to a style I use (they are previously my comments)

**Changelog**

:cl: DangerRevolution
- add: Nanotrasen reports new Cyborg lawsets present from IonStorms.

### CODE REVIEW

File: Resources/Prototypes/DeltaV/siliconlaws.yml
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/siliconlaws.yml:678]: noob these dont have loc

Comment (DangerRevolution) [reply Resources/Prototypes/DeltaV/siliconlaws.yml:678]: noob moment fml

Comment (DangerRevolution) [reply Resources/Prototypes/DeltaV/siliconlaws.yml:678]: gamed hard with web editing; thanks Delta :)

**Comment by DangerRevolution** [comment]
test fail not related to me

**Review by deltanedas — DISMISSED** [review]
lgtm merging once admeme approve

**Comment by therealDLondon** [comment]
These look fine to me, borg laws are always great

**Review by therealDLondon — APPROVED** [review]

**Review by deltanedas — APPROVED** [review]

**Pull request merged**
